🚚 Migrate database population scripts

davikch opened the pull request:
Other changes include: minor env.example formatting, move scripts from separate folders to one common folder, refactor god awful category code

Oh and how could I forgot renaming downscale.hs to downscale.sh

BeaverNotACat reviewed (approved):
https://youtu.be/LabIat9t-uY

davikch committed:
🚚 Migrate database population scripts

Other changes include: minor env.example formatting, move scripts from separate folders to one common folder, refactor god awful category code
